recipes/r-smoothr/meta.yaml

```
@@ -0,0 +1,91 @@
+{% set version = '1.0.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-smoothr
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/smoothr_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/smoothr/smoothr_{{ version }}.tar.gz
+  sha256: 580b150009934475173a6fc917020bc423934103021a8d3e6b6620d2b93fc5e9
+
+build:
+  skip: true  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+# Suggests: covr, knitr, lwgeom, methods, rmarkdown, sp, testthat
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+  host:
+    - r-base
+    - r-sf
+    - r-terra >=1.6_3
+    - r-units
+  run:
+    - r-base
+    - r-sf
+    - r-terra >=1.6_3
+    - r-units
+
+test:
+  requires:
+    - r-testthat
+    - r-sp
+  files:
+    - testthat.R
+    - testthat/
+  commands:
+    - $R -e "library('smoothr')"           # [not win]
+    - $R -f testthat.R                     # [not win]
+    - "\"%R%\" -e \"library('smoothr')\""  # [win]
+
+
+about:
+  home: https://strimas.com/smoothr/, https://github.com/mstrimas/smoothr
+  license: GPL-3.0-or-later
```

cran meta data says: # License: GPL-3
```suggestion
  license: GPL-3.0-only
```